webapp/middlewares/cookieParser.js

```diff
@@ -0,0 +1,7 @@
+import cookieParser from "cookie-parser"
```

Comment:
Please change usage of 3rd party library to native implementation. Cookies are stored in `req.headers.cookies`